Codeinwp/neve
# Refactor/conditional headers

### Summary
Adds changes to account for the new version of the Conditional Headers control [here](https://github.com/Codeinwp/neve-pro-addon/pull/1477/).


### Will affect visual aspect of the product

NO

### Test instructions


- Use this PR: https://github.com/Codeinwp/neve-pro-addon/pull/1477/
- Conditional headers will not be available without the new version of pro, so they should still not show up in customizer using the master branch version.
